health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -2361,23 +2361,37 @@
 
 const enrichTargetForProject = (project: any, codesTargetMapping: any, boundaryCode: any) => {
   if (codesTargetMapping?.[boundaryCode] && Object.keys(codesTargetMapping[boundaryCode]).length > 0) {
-    let targets = [];
-    for (const key in codesTargetMapping?.[boundaryCode]) {
-      let targetNo = parseInt(codesTargetMapping?.[boundaryCode][key]);
-      let beneficiaryType = key;
-      targets.push({
-        beneficiaryType: beneficiaryType,
-        targetNo: targetNo,
-      });
+    let targets: any[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'targets' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)